src/backend/winit/mod.rs

```diff
@@ -302,12 +302,19 @@ impl WinitGraphicsBackend {
         Ok(())
     }
 
-    /// Retrieve the buffer age of the current backbuffer of the window
-    pub fn buffer_age(&self) -> usize {
+    /// Retrieve the buffer age of the current backbuffer of the window.
+    ///
+    /// This will only return a meaningful value, if this `WinitGraphicsBackend`
+    /// is currently bound (by previously calling [`WinitGraphicsBackend::bind`]).
+    ///
+    /// Otherwise and on error this function returns `None`.
```

Comment:
So there can be errors other than the fact that the backend is not currently bound? Would they be worth reporting?

Reply:
They are usually flukes, no other errors are defined by the function, but is defined as possible to fail.